evaluation/README.md

```diff
@@ -24,3 +24,18 @@ This example uses a very small, fake dataset meant to show working code. Suggest
 
 1. A groundtruth evaluation dataset relevant to the application's domain
 2. A large search dataset such as [MS Marco](https://microsoft.github.io/msmarco/)
+
+## E2E Samples
```

Comment:
nit: we don't define the acronym E2E

some readers may not understand this means End to End sample

Reply:
also, could we rename `e2e_samples` where it exists in our directory to something else? maybe `chunking_evaluation`?

Reply:
Renamed E2E to End-to-End and rename directory `e2e_samples/chunking` to `chunking_evaluation`